### PR TITLE
Set the error tag on 500 responses

### PR DESCRIPTION
**Pull request by jeremiahishere**

### CODE REVIEW

File: lib/rack/instrumentation/tracer.rb [hunk 1]
```diff
@@ -30,6 +30,7 @@ def trace_request(env, scope)
 
         app.call(env).tap do |status_code, _headers, _body|
           span.set_tag('http.status_code', status_code)
+          span.set_tag('error', true) if status_code >= 500 && status_code < 600
```
Comment (josh-lauer) [inline lib/rack/instrumentation/tracer.rb]: I think something like `span.set_tag('error', true) if (500..599).include?(status_code)` is maybe a bit more idiomatic.

**Review by josh-lauer — APPROVED** [review]
LGTM

**Comment by jeremiahishere** [comment]
Ready to merge now.

**Comment by josh-lauer** [comment]
It's built and pushed to RubyGems.

**Pull request merged**